src/org/zaproxy/zap/view/StandardFieldsDialog.java

```diff
@@ -1159,6 +1159,43 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		this.incTabOffset(tabIndex);
 	}
 	
+	/**
+	 * Add a 'node select' field which provides a button for showing a Node Select Dialog and a 
+	 * non editable field for showing the node selected
+	 * 
+	 * TODO: add version
```

Comment:
Sorry, I was not clear, it should still have the tag, e.g. `@since TODO add version`.

Reply:
no problem